install.sh

```diff
@@ -507,7 +511,7 @@ download_binary() {
         # Since Binary and Hash are zipped together, check if TMP_ZIP already exists
         if ! [ -f ${TMP_ZIP} ]; then
             info "Downloading K3s artifact ${GITHUB_PR_URL}"
-            curl -o ${TMP_ZIP} -H "Authorization: Bearer $GITHUB_TOKEN" -L ${GITHUB_PR_URL}
+            curl -o -f ${TMP_ZIP} -H "Authorization: Bearer $GITHUB_TOKEN" -L ${GITHUB_PR_URL}
```

Comment:
I think the -f needs to before -o, as TMP_ZIP is the value of the -o flag; you can't put another flag in between.

Reply:
Nice catch, that is a typo left over from debugging issue